### PR TITLE
Traffic law in the United States

### PR DESCRIPTION
**Pull request by eadamsintel**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

I am doing the instruct lab at the Red Hat Summit. I hope that my contribution helps!

- ...
- ...
- ...

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Comment by eadamsintel** [comment]
https://github.com/instructlab/taxonomy/issues/811

File: knowledge/laws/traffic_laws_us/qna.txt
Comment (jjasghar) [inline knowledge/laws/traffic_laws_us/qna.txt:1]: Please change the name to `qna.yaml`

File: knowledge/laws/traffic_laws_us/qna.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+created_by:
```
Comment (jjasghar) [inline knowledge/laws/traffic_laws_us/qna.txt:1]: Please add your GitHub username here.

File: knowledge/laws/traffic_laws_us/qna.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  question: In the United States how are road signs used?
+- answer: |
+  49 states, the District of Columbia and the five inhabited territories have passed laws requiring seat belt use by at least all occupants of the front seat. 
+  question: What states require seat belt use?
+- answer: |
+  Overtaking, usually called "passing", is legal on all four or more lane roads and on most two-lane roads with sufficient sight distance.
+  question: Is overtaking legal?
+- answer: |
+  Speed limits are set by each state or territory, as well as counties or municipalities, on the roads within their jurisdiction. The maximum speed limit on rural two-lane roads ranges from 50 mph (80 km/h) in parts of the northeast to 75 mph (120 km/h) in parts of Texas. On rural Interstate Highways and other freeways, the speed limit ranges from 60 mph (96 km/h) in Hawaii to 85 mph (136 km/h) in parts of Texas. All roads in the United States have a speed limit, but it is not always posted (especially in rural areas).
```
Comment (jjasghar) [inline knowledge/laws/traffic_laws_us/qna.txt:17]: Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please make these changes then we can run the linter.

**Comment by eadamsintel** [comment]
I hopefully fixed this. I don't normally make edits directly from the GitHub UI so hopefully this went through OK.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please check the files on this PR for the linting issues you're missing.

**Comment by jjasghar** [comment]
Closing per combined PR here: https://github.com/instructlab/taxonomy/pull/1151

**Pull request closed without merging**